### ISSUE TITLE
min_bid is a string while max_bid is a float

### ISSUE THREAD

**Issue by zachliu** [MediaMath/t1-python#133]
```python
>>> from terminalone import T1
>>> T1_SDK = T1(T1_USERNAME, T1_PASSWORD, T1_APIKEY)
>>> st = T1_SDK.get("strategies", <STRATEGY_ID>)
>>> type(st.min_bid)
<class 'str'>
>>> type(st.max_bid)
<class 'float'>
```

**Comment by FodT** [comment]
It looks like `min_bid` is actually missing from the entity model for `Strategy`.  Thanks for flagging this, @zachliu

**Comment by FodT** [comment]
resolved by #135
